### ISSUE TITLE
Failure to establish SSTP VPN between Fedora 31 client and Windows Server 2008

### ISSUE THREAD

**Issue by szpak** [enaess/network-manager-sstp#32]
We've got an error report in the Fedora Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1778519

The short version is:
> In the simplest case, the WiFi connection CJM-5 is established and I try "sstpc --warn-cert terracina.tclc.org" and I see /var/log/messages says: ethtool: autonegotiation is unset or enabled, the speed and duplex are not writable.

@enaess Do you know what could a reason? More details are in the linked issue.

**Comment by enaess** [comment]
Interesting, I will have to take a look come this weekend. By the way, I am looking into adding EAP support. Do you know if the eap-tls patches are applied on the RedHat version of pppd? They are on Ubuntu. 
Regards,
- Eivind

    På tirsdag 3. desember 2019, 12:48:06 PST skrev Marcin Zajączkowski <notifications@github.com> følgende:  
 
 
We've got an error report in the Fedora Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1778519

The short version is:


In the simplest case, the WiFi connection CJM-5 is established and I try "sstpc --warn-cert terracina.tclc.org" and I see /var/log/messages says: ethtool: autonegotiation is unset or enabled, the speed and duplex are not writable.


@enaess Do you know what could a reason? More details are in the linked issue.

**Comment by szpak** [comment]
> Do you know if the eap-tls patches are applied on the RedHat version of pppd?

I've checked the RPM specification in Fedora and they've seen to be [applied](https://src.fedoraproject.org/rpms/ppp/blob/master/f/ppp-2.4.7-eaptls-mppe-1.102.patch) long time ago (~1 year) in Fedora, so it should be fine

**Comment by szpak** [comment]
@enaess Have you been able to take a closer look at that?

**Comment by enaess** [comment]
Did manage to spare some time to do some UI redesign. Can't promisse a date, but work is in progress 




Get Outlook for Android







On Fri, Dec 20, 2019 at 3:10 PM -0600, "Marcin Zajączkowski" <notifications@github.com> wrote:












@enaess Have you been able to take a closer look at that?

**Comment by enaess** [comment]
@szpak do you have an update on this? Fedora 31 is no longer maintained. Is this still a problem on newer versions of Fedora. By the way, I've run into the same problem with ethtool when using OpenVPN. I think there was a different patch to the udev/systemd stuff that fixed that ...

**Comment by szpak** [comment]
Closing as F31 has reached EOL.
